--- a/backend/CMakeLists.txt
+++ b/backend/CMakeLists.txt
@@ -5,6 +5,8 @@
   URL_HASH
     "SHA512=6098efa217d6c5fcdf6d6cbc6051020e728bdbf6190ca3d5be90e9fc382b8d666f1649506f7b4b70abff6084481799eba0fb0a44ad82a0bedfab98ee855e8d22"
   PATCH_COMMAND
+    patch llvm/include/llvm/Demangle/MicrosoftDemangleNodes.h ${CMAKE_CURRENT_LIST_DIR}/MicrosoftDemangleNodes.h.patch
+  COMMAND
     ln -sf ${CMAKE_CURRENT_LIST_DIR}/clang/Attr.td clang/include/clang/Basic/Attr.td
   COMMAND
     ln -sf ${CMAKE_CURRENT_LIST_DIR}/clang/Preprocessor.h clang/include/clang/Lex/Preprocessor.h
