diff --git a/backend/python/tapa/steps/analyze.py b/backend/python/tapa/steps/analyze.py
index 97610aa..1300c01 100644
--- a/backend/python/tapa/steps/analyze.py
+++ b/backend/python/tapa/steps/analyze.py
@@ -151,12 +151,7 @@ def find_tapacc_cflags(
     _logger.info('added vendor include path `%s`', vendor_path)
 
   # Add system include files to tapacc cflags
-  system_includes = ['-stdlib=libc++']
-  system_includes.extend(
-      ['-isystem', f"/usr/lib/llvm-{match[1]}/include/c++/v1/"])
-  system_includes.extend(
-      ['-isystem', f"/usr/include/clang/{match[1]}/include/"])
-  system_includes.extend(['-isystem', f"/usr/lib/clang/{match[1]}/include/"])
+  system_includes = []
 
   return (cflags[:] + ('-I', tapa_include) + vendor_include_paths,
           tuple(system_includes))
