diff --git a/backend/python/tapa/steps/analyze.py b/backend/python/tapa/steps/analyze.py
index 97610aa..fe7c653 100644
--- a/backend/python/tapa/steps/analyze.py
+++ b/backend/python/tapa/steps/analyze.py
@@ -151,14 +151,10 @@ def find_tapacc_cflags(
     _logger.info('added vendor include path `%s`', vendor_path)
 
   # Add system include files to tapacc cflags
-  system_includes = ['-stdlib=libc++']
-  system_includes.extend(
-      ['-isystem', f"/usr/lib/llvm-{match[1]}/include/c++/v1/"])
-  system_includes.extend(
-      ['-isystem', f"/usr/include/clang/{match[1]}/include/"])
-  system_includes.extend(['-isystem', f"/usr/lib/clang/{match[1]}/include/"])
-
-  return (cflags[:] + ('-I', tapa_include) + vendor_include_paths,
+  system_includes = []
+  extra_flags = ('-nostdinc', '-nostdinc++')
+
+  return (cflags[:] + ('-I', tapa_include) + vendor_include_paths + extra_flags,
           tuple(system_includes))
 
 
