--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -259,13 +259,14 @@
 )
 include(CPack)

-enable_testing()
-
-include(GoogleTest)
-
-add_executable(buffer_test src/frt/buffer_test.cpp)
-target_link_libraries(buffer_test frt GTest::gtest_main)
-gtest_discover_tests(buffer_test)
-
-add_subdirectory(tests/hbm)
-add_subdirectory(tests/xdma)
+include(CTest)
+if(BUILD_TESTING)
+  include(GoogleTest)
+
+  add_executable(buffer_test src/frt/buffer_test.cpp)
+  target_link_libraries(buffer_test frt GTest::gtest_main)
+  gtest_discover_tests(buffer_test)
+
+  add_subdirectory(tests/hbm)
+  add_subdirectory(tests/xdma)
+endif()
